db/test.sql

```diff
@@ -6,4 +6,12 @@ SELECT  reviews.review_id, comments.review_id, owner, title, category, review_im
   JOIN comments
   ON reviews.review_id = comments.review_id
   GROUP BY reviews.review_id
- ORDER BY created_at DESC;
\ No newline at end of file
+ ORDER BY created_at DESC;
+
+ SELECT  reviews.review_id, reviews.owner, reviews.title, reviews.category, review_img_url, reviews.created_at, reviews.votes, reviews.designer,  COUNT (comments.review_id)::INT AS "comment_count"
+  FROM reviews
+  JOIN comments
+  ON reviews.review_id = comments.review_id
+  GROUP BY reviews.review_id
+  ORDER BY created_at DESC
+  ;`
```

Comment:
Its absolutly fine to have a test.sql file here but i would recommend adding this file specifically to your git ignore file.

Reply:
Added to gitignore, thanks.